data/functions.sql

```diff
@@ -122,7 +122,8 @@ CREATE OR REPLACE FUNCTION mz_calculate_road_level(highway_val text, railway_val
 RETURNS SMALLINT AS $$
 BEGIN
     RETURN (
-        CASE WHEN highway_val IN ('motorway', 'trunk', 'primary', 'motorway_link') THEN 8
+        CASE WHEN highway_val IN ('motorway', 'trunk', 'motorway_link') THEN 8
+             WHEN highway_val IN ('primary') THEN 9
```

Comment:
All roads need to be visible, else gaps are visible. There are places where a primary road down-grades to a secondary which will show as gaps. I looked at the road network at z8 with primary included, and it was far too busy. If you'd like this changed, please open a new issue.